README.md

```diff
@@ -42,8 +49,28 @@ Install all dependencies and the project:
 poetry install
 ```
 
-Start the server in simulation mode:
+## Start the server
+
+### Either start the server directly on your host
 
 ```
-python3 -m nfv_test_api.main --config config.yaml --simulate
-```
\ No newline at end of file
+python3 -m nfv_test_api.main --config config.yaml
```

Comment:
I would add a line saying where the api can be accessed (and show the path to the swagger)